tests/basic/backup_test.go

```diff
@@ -1936,3 +1950,31 @@ func getBackupUID(orgID, backupName string) string {
 			orgID, backupName, err))
 	return backupUID
 }
+
+func validateBackupCluster() bool {
+	labelSelectors := make(map[string]string)
+	ns := backup.GetPxBackupNamespace()
+	pods, err := core.Instance().GetPods(ns, labelSelectors)
+	if err != nil {
+		logrus.Warnf("Unable to fetch the pods from backup namespace\n Error : [%v]\n",
+			err)
+		return false
+	}
+	for _, pod := range pods.Items {
+		matched, _ := regexp.MatchString("^pxcentral-post-install-hook", pod.GetName())
+		if !matched {
```

Comment:
@snigdha-px you will have to validate whether pxcentral-post-install-hook pod is in completed state or not. This is the confirmation for px-backup successful installation.

Reply:
Done